src/Atmos/Model/source.jl

```diff
@@ -38,33 +38,37 @@ struct GeostrophicForcing{FT} <: Source
   u_geostrophic::FT
   v_geostrophic::FT
 end
-function atmos_source!(s::GeostrophicForcing, m::AtmosModel, source::Vars, state::Vars, aux::Vars, t::Real)
-  u = state.ρu / state.ρ
+function atmos_source!(s::GeostrophicForcing, atmos::AtmosModel, source::Vars, state::Vars, aux::Vars, t::Real)
   u_geo = SVector(s.u_geostrophic, s.v_geostrophic, 0)
-  source.ρu -= state.ρ * s.f_coriolis * (u - u_geo)
+  ẑ = vertical_unit_vector(atmos.orientation, aux)
+  fkvector = s.f_coriolis * ẑ
+  source.ρu -= fkvector × (state.ρu .- state.ρ*u_geo)
 end
 
 """
-  RayleighSponge{FT} <: Source
+    RayleighSponge{FT} <: Source
+
 Rayleigh Damping (Linear Relaxation) for top wall momentum components
 Assumes laterally periodic boundary conditions for LES flows. Momentum components
 are relaxed to reference values (zero velocities) at the top boundary.
 """
 struct RayleighSponge{FT} <: Source
-  "Domain maximum height [m]"
-  zmax::FT
-  "Vertical extent at with sponge starts [m]"
-  zsponge::FT
-  "Sponge Strength 0 ⩽ c_sponge ⩽ 1"
-  c_sponge::FT
+  "Maximum domain altitude (m)"
+  z_max::FT
+  "Altitude at with sponge starts (m)"
+  z_sponge::FT
+  "Sponge Strength 0 ⩽ α_max ⩽ 1"
+  α_max::FT
+  "Relaxation velocity components"
+  u_relaxation::SVector{3,FT}
```

Comment:
Is `u_relaxation` global everywhere?

Reply:
Well, it's only used in the sponge layer, which seems fine in a box but it's a good question for the sphere.

Reply:
On the sphere, we'd want this `u_relaxation` to be the zonal mean (i.e., mean over longitudes)--however this is best implemented.

Reply:
okay, we may need some extra object to handle that. we can add that later.